pkgs/applications/graphics/vengi-tools/test-voxconvert-roundtrip.nix

```diff
@@ -1,15 +0,0 @@
-{ stdenv
-, vengi-tools
-}:
-
-stdenv.mkDerivation {
-  name = "vengi-tools-test-voxconvert-roundtrip";
-  meta.timeout = 10;
-  buildCommand = ''
-    ${vengi-tools}/bin/vengi-voxconvert ${vengi-tools}/share/vengi-voxedit/chr_knight.qb chr_knight.vox
-    ${vengi-tools}/bin/vengi-voxconvert chr_knight.vox chr_knight.qb
-    ${vengi-tools}/bin/vengi-voxconvert chr_knight.qb chr_knight1.vox
-    diff chr_knight.vox chr_knight1.vox
```

Comment:
We're loosing something here. The roundtrip test tested only the `.vox` format, but it tested whether the file was the same after converting it back. We don't have that anymore in your new test. Is there any chance you can change your parametrised test in such a way that it is a roundtrip test for every format? (Or at least a roundtrip test for some formats and a simple test as you proposed for all the others?)

Reply:
Oh, I see, the roundtrip test was failing previously. Why?

I'm not much into the topic and can't judge whether it makes sense to demand a roundtrip test. If it's too much effort, a smoke test as you proposed is great.

Reply:
I already contacted the author about the roundtrip failure. It looks like stuff is getting reordered on each iteration. If it gets fixed, I'll add the test back

Reply:
Ok, sounds like it could be fixed if the test data is already ordered maybe?

That's fine for me :smile: if the communication about the roundtrip test failure is public, feel free to link an issue or so in a comment in the derivation test attribute, if not no worries.

Reply:
> Ok, sounds like it could be fixed if the test data is already ordered maybe?

Unfortunately no. That was the case on the previous version: notice that the roundtrip test did _two_ conversions to let the order stabilize. Now the order appears to be indefinitely unstable.

I'll open and link to a ticket in a bit